winui/Kanban/swimlane.md

```diff
@@ -0,0 +1,220 @@
+---
+layout: post
+title: Swimlane support in WinUI Kanban control | Syncfusion
+description: Learn here all about the swimlane support and its customization in Syncfusion WinUI Kanban (SfKanban) control.
+platform: winui
+control: SfKanban
+documentation: ug
+---
+
+# Swim lanes in WinUI Kanban Control
```

Comment:
Swimlane in WinUI Kanban (SfKanban) control

Reply:
Changed.